### DISCUSSION TITLE
Is there a version compatibility table for Docker, Docker Compose, and Docker Harbor?

### DISCUSSION THREAD

**Discussion by 02darlingX** [goharbor/harbor#20663] [Q&A]
Dear experts，
Is there a version compatibility table for Docker, Docker Compose, and Docker Harbor?
When downloading Harbor, I'm not sure whether they are compatible.

**Comment by MinerYang** [comment]
Please refer to https://goharbor.io/docs/2.11.0/install-config/installation-prereqs/

**Reply by 02darlingX** [reply]
thank you
